packages/abi-gen-templates/partials/callAsync.handlebars

```diff
@@ -34,6 +41,11 @@ async callAsync(
     // tslint:enable boolean-naming
     return result;
 },
+
+/**
+ * Returns the ABI encoded transaction data
```

Comment:
`Returns the ABI encoded transaction data that would be sent in the Ethereum tx calling this method. This encoded tx data can then be sent to a separate signing service or can be used to create a 0x transaction (see protocol spec for more details).`